po/common/en.po

```diff
@@ -2507,6 +2507,10 @@ msgctxt "site_name"
 msgid "Open Food Facts"
 msgstr "Open Food Facts"
 
+msgctxt "logo_site_name"
+msgid "Logo Open Food Facts"
```

Comment:
Could you change it to "Open Food Facts logo"?

Reply:
yes, sure. You want "L" to be small?

Reply:
yes, small L.

You can keep this PR for the logo, but I think the other bug (the underlined links) should be done in another way, and moved to another PR.

Reply:
> I think it would be better to split this into 2 PRs. The alt text for the logo is easy, but the underline links would probably be better done in another way: #5074
> 
> If we split it, then we can merge each PR independently.

Yes, it'll be better. I will open PR for each issue separately now.